<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetForm</class>
 <widget class="QWidget" name="PetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-radius:3px;
background-color:qlineargradient(spread:pad,x1:0,y1:1,x2:1,y2:0,stop:0 rgba(255,200,177,80%),stop:0.6 rgba(155,145,255,70%),stop:1 rgba(80,220,250,70%));




</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="PetIco">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>69</y>
        <width>57</width>
        <height>57</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>57</width>
        <height>57</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>57</width>
        <height>57</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgb(255, 255, 255);
border-radius:15px;
border:2px outset rgba(140,80,255,75%);
}
QLabel:hover{
border:2px outset rgba(255,80,144,80%);
border-color:deeppink;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="wgIco">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);
border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="mgIco">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="fyIco">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="sdIco">
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="mkIco">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="mzIco">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="sbIco">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="bjIco">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>131</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:3px;
border:1px outset rgb(160,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="status_3">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>64</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px outset rgb(220,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="status_2">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>86</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px outset rgb(220,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="status_1">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>108</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border:2px outset rgb(220,120,33);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="equip_1">
      <property name="geometry">
       <rect>
        <x>76</x>
        <y>36</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
border:2px outset rgb(80,220,100);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="equip_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>36</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
border:2px outset rgb(80,220,100);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="equip_3">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>36</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
border:2px outset rgb(80,220,100);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="blood">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>36</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
border:2px outset rgb(80,220,100);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="skill_1">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>65</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font: 85 9pt &quot;微软雅黑&quot; ;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);

}
</string>
      </property>
      <property name="text">
       <string>一号技能:</string>
      </property>
     </widget>
     <widget class="PushLabel" name="skill_2">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>81</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 85 9pt &quot;微软雅黑&quot;;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);</string>
      </property>
      <property name="text">
       <string>二号技能:</string>
      </property>
     </widget>
     <widget class="PushLabel" name="skill_3">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>97</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 85 9pt &quot;微软雅黑&quot;;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);
</string>
      </property>
      <property name="text">
       <string>三号技能:</string>
      </property>
     </widget>
     <widget class="PushLabel" name="skill_4">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>113</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 85 9pt &quot;微软雅黑&quot;;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);
</string>
      </property>
      <property name="text">
       <string>四号技能:</string>
      </property>
     </widget>
     <widget class="TLabel" name="name">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font: 85 9pt &quot;微软雅黑&quot; ;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);

}
QLabel text{
	fill-color:transparent;
	stroke:1px red;
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>宠【1】</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="TLabel" name="Intimacy">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>36</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font: 85 10pt &quot;微软雅黑&quot; ;
color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);

}
</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>亲密度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="TLabel" name="level">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: transparent;
border:0px outset rgb(160,120,33);
font: 85 9pt &quot;微软雅黑&quot; ;

</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Lv:000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Probar" name="hp" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>18</y>
        <width>160</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px outset rgb(220,150,0);
background-color: transparent;</string>
      </property>
     </widget>
     <widget class="QLabel" name="Hint">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>83</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border:0px outset rgb(160,120,33);
color: rgb(236, 236, 236);</string>
      </property>
      <property name="text">
       <string>阵亡</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="PushLabel" name="propety_1">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>64</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:11px;
border:2px outset rgb(0,150,255);
background-color: rgb(255, 255, 255);

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="PushLabel" name="propety_2">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>109</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:11px;
border:2px outset rgb(0,150,255);
background-color: rgb(255, 255, 255);

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>status_1</zorder>
     <zorder>status_2</zorder>
     <zorder>status_3</zorder>
     <zorder>PetIco</zorder>
     <zorder>equip_3</zorder>
     <zorder>equip_1</zorder>
     <zorder>wgIco</zorder>
     <zorder>blood</zorder>
     <zorder>equip_2</zorder>
     <zorder>skill_1</zorder>
     <zorder>skill_2</zorder>
     <zorder>skill_3</zorder>
     <zorder>skill_4</zorder>
     <zorder>name</zorder>
     <zorder>Intimacy</zorder>
     <zorder>level</zorder>
     <zorder>hp</zorder>
     <zorder>mgIco</zorder>
     <zorder>fyIco</zorder>
     <zorder>sdIco</zorder>
     <zorder>mkIco</zorder>
     <zorder>mzIco</zorder>
     <zorder>sbIco</zorder>
     <zorder>bjIco</zorder>
     <zorder>propety_1</zorder>
     <zorder>propety_2</zorder>
     <zorder>Hint</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TLabel</class>
   <extends>QLabel</extends>
   <header location="global">tlabel.h</header>
  </customwidget>
  <customwidget>
   <class>PushLabel</class>
   <extends>QLabel</extends>
   <header location="global">pushlabel.h</header>
  </customwidget>
  <customwidget>
   <class>Probar</class>
   <extends>QWidget</extends>
   <header location="global">probar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
